src/js/UI-remove-task.js

```diff
@@ -0,0 +1,5 @@
+export default class RemoveActivities {
```

Comment:
- [ ] Working on new milestones should not remove previous functionality added, let's make sure that the clear all completed button keeps working as expected.

> Implement a function for the "Clear all completed" button (use filter() method).

![no-completed](https://user-images.githubusercontent.com/79114233/150185779-e8ce873b-1b3a-4006-9049-5aef8538d5f4.gif)